python/tvm/relay/op/dyn/nn/_nn.py

```diff
@@ -47,12 +61,12 @@ def compute_upsampling(attrs, inputs, out_dtype):
 
 # upsampling
 @script
-def _upsampling_shape_func(dshape, scale_h, scale_w, height_axis, width_axis, channel_axis):
```

Comment:
Why remove channel axis here?

Reply:
It's an unnecessary argument. We've already copied the channel axis in the loop, and since we don't modify that axis, we don't need to recopy it